nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 41 Line 1_
```python
users()
```
Is this supposed to have some parameter like ```users(sort="name")```?